packages/runtime/src/runtimes/react/controls/slot.tsx

```diff
@@ -166,7 +166,7 @@ function SlotPlaceholder({ control }: SlotPlaceholderProps): JSX.Element {
   )
 }
 
-function pollBoxModel({
+export function pollBoxModel({
```

Comment:
This is not what I will be doing in the final version. This PR is simply here for use to start the conversation.